test-helpers/src/behaviors/balance.ts

```diff
@@ -18,22 +18,22 @@ function base(execute: Execute) {
             },
           },
         },
-        {
-          name: 'unknown endpoint',
-          testData: {
-            id: jobID,
-            data: {
-              endpoint: 'not_real',
-              result: [
-                {
-                  address: '35ULMyVnFoYaPaMxwHTRmaGdABpAThM4QR',
-                  coin: 'btc',
-                  chain: 'mainnet',
-                },
-              ],
-            },
-          },
-        },
+        // {
```

Comment:
Why is this test being disabled?

When I checkout to the branch and run the test it passes for me? Was this an issue when running in CI?

I'm seeing a different issue with the balance decimal place conversion that I will open an issue for.

Reply:
It was related to a default routing to an endpoint instead of error routing. Removing it.